Swift/Controllers/XMLConsoleController.h

```diff
@@ -12,13 +12,15 @@
 #include <boost/signals2.hpp>
 
 #include <Swiften/Base/SafeByteArray.h>
+#include <Swiften/Client/Client.h>
```

Comment:
If the predeclaration below is sufficient, then there's no need to include `Client.h` here, right? If not, the predeclaration is not needed anymore.